mindmeld/converter/rasa.py

```diff
@@ -0,0 +1,505 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2015 Cisco Systems, Inc. and others.  All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This module contains the Rasacoverter class used to convert Rasa projects
+into Mindmeld projects"""
+
+from keyword import iskeyword
+import re
+import os
+import copy
+import logging
+import yaml
+
+from mindmeld.converter.converter import Converter
+
+logger = logging.getLogger(__name__)
+
+
+class RasaConverter(Converter):
+    """The class is a sub class of the abstract Converter class. This class
+    contains the methods required to convert a Rasa project into a MindMeld project
+    """
+    def __init__(self, rasa_project_directory, mindmeld_project_directory):
+        if os.path.exists(os.path.dirname(rasa_project_directory)):
+            self.rasa_project_directory = rasa_project_directory
+            self.mindmeld_project_directory = mindmeld_project_directory
+        else:
+            raise FileNotFoundError
+
+    def _create_intents_directories(self, mindmeld_project_directory, intents):
+        for intent in intents:
```

Comment:
Please state the design decision here of adding only one domain (since Rasa does not have domain classification). Moreover,  make the string `"/domains/general/"` into a constant and name that meaningfully.